### PR TITLE
Integrations detail page objects

### PR DESCRIPTION
**Pull request by lvydra**

### CODE REVIEW

**Comment by Stefan365** [comment]
LGTM

File: ui-tests/src/test/java/io/syndesis/qe/pages/integrations/detail/IntegrationDetailPageFactory.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
+package io.syndesis.qe.pages.integrations.detail;
+
+public class IntegrationDetailPageFactory {
+
+	public IntegrationDetailPage getDetailPage(String integratinStatus) {
```
Comment (dsimansk) [inline ui-tests/src/test/java/io/syndesis/qe/pages/integrations/detail/IntegrationDetailPageFactory.java]: Typo in `integratinStatus`. But this is interesting approach actually, I need to think about it a bit :)

File: ui-tests/src/test/java/io/syndesis/qe/pages/integrations/detail/IntegrationDetailPage.java [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+	public boolean validate() {
+		return getRootElement().is(visible);
+	}
+
+	public String getIntegrationName() {
+		return this.getElementText(Element.TITLE);
+	}
+
+	public void deleteIntegration() {
+		this.clickButton("Delete Integration");
```
Comment (dsimansk) [inline ui-tests/src/test/java/io/syndesis/qe/pages/integrations/detail/IntegrationDetailPage.java]: Shouldn't we rather reuse that nice enum like class `IntegrationDetailActions`?

File: ui-tests/src/test/java/io/syndesis/qe/pages/integrations/detail/IntegrationDetailActions.java [hunk 3]
```diff
@@ -0,0 +1,8 @@
+package io.syndesis.qe.pages.integrations.detail;
+
+public class IntegrationDetailActions {
```
Comment (dsimansk) [inline ui-tests/src/test/java/io/syndesis/qe/pages/integrations/detail/IntegrationDetailActions.java]: Enum maybe?

File: ui-tests/src/test/java/io/syndesis/qe/pages/integrations/detail/IntegrationDetailPage.java [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+	}
+
+	public String getStatus() {
+		return this.getElementText(Element.STATUS);
+	}
+
+	public void performAction(String action) {
+		if (this.actionsSet.contains(action)) {
+			this.clickButton(action);			
+		}
```
Comment (dsimansk) [inline ui-tests/src/test/java/io/syndesis/qe/pages/integrations/detail/IntegrationDetailPage.java]: What about some nice error in else branch of if?

File: ui-tests/src/test/java/io/syndesis/qe/pages/integrations/detail/IntegrationDetailPage.java (hunk 2 above)
Comment (lvydra) [reply ui-tests/src/test/java/io/syndesis/qe/pages/integrations/detail/IntegrationDetailPage.java]: Good idea, done.

File: ui-tests/src/test/java/io/syndesis/qe/pages/integrations/detail/IntegrationDetailPage.java (hunk 4 above)
Comment (lvydra) [reply ui-tests/src/test/java/io/syndesis/qe/pages/integrations/detail/IntegrationDetailPage.java]: Same as previous, done.

File: ui-tests/src/test/java/io/syndesis/qe/pages/integrations/detail/IntegrationDetailActions.java (hunk 3 above)
Comment (lvydra) [reply ui-tests/src/test/java/io/syndesis/qe/pages/integrations/detail/IntegrationDetailActions.java]: I'm not sure if we would really benefit from it.

**Pull request merged**